### ISSUE TITLE
[med] JAWS+Chrome: Chat: Currently selected chat does not announce as current

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5684]
## Action Performed:
Prerequisites:
- User has created personal account and is logged into same.
- There is at least a few (4+) chat session listed in the 'Chats' panel on the left.

1. Using JAWS+Chrome, open https://staging.new.expensify.com
2. Tab to focus the currently active chat session in the "Chats" panel (indicated by a darker shading of grey and the name matching the main chat window title) and note how it is announced.

## Expected Result:
The chat should announce its name, role, and currently selected state. e.g. "Button, [email, name, phone] [last message], selected/current page".

## Actual Result:
The button announces it's title and latest message without the selected state or button role (see issue 5179534): "[email, name, phone] [last message]".

HTML:
`<div tabindex="0" class="css-1dbjc4n r-1loqt21 r-1otgn73 r-1i6wzkk r-lrvibr" style="align-items: center; background-color: rgb(236, 236, 236); flex-direction: row; justify-content: space-between; padding-left: 20px; padding-right: 20px; text-decoration-line: none; transition-duration: 0s; -webkit-box-align: center; -webkit-box-orient: horizontal; -webkit-box-direction: normal; -webkit-box-pack: justify;"><div class="css-1dbjc4n r-13awgt0 r-16y2uox" style="height: 64px; justify-content: center; min-width: 0px; padding-bottom: 12px; padding-top: 12px; text-decoration-line: none; -webkit-box-pack: center;"><div class="css-1dbjc4n" style="align-items: center; flex-direction: row; -webkit-box-align: center; -webkit-box-orient: horizontal; -webkit-box-direction: normal;">...</div>`

## Workaround:
No workaround

## Area issue was found in:
Chat page

## Failed WCAG checkpoints
4.1.2

## User impact:
The screen reader users will not be able to tell which chat is the currently active one in the main content area when going through the list like non-sight-impaired users can.

## Suggested resolution:
Use the aria-current="true" attribute to indicate the current element in a set of chats.
https://www.w3.org/TR/wai-aria-1.2/#aria-current

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/136223062-a277042f-1e1b-4862-b7c1-25f1cf6c7fbe.mp4

![Bug5197784_JAWS_Chrome-Chat-Currently_selected_chat_does_not_announce_same_in_list](https://user-images.githubusercontent.com/88733897/136223069-a0b074a3-6fab-41c1-8878-f90dac00bfb4.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
